src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/TestSession.cs

```diff
@@ -272,6 +272,15 @@ public bool StopTestSession()
     /// <inheritdoc/>
     [Obsolete("This API is not final yet and is subject to changes.", false)]
     public bool StopTestSession(ITestSessionEventsHandler eventsHandler)
+    {
+        return StopTestSession(options: null, eventsHandler);
+    }
+
+    /// <inheritdoc/>
+    [Obsolete("This API is not final yet and is subject to changes.", false)]
+    public bool StopTestSession(
```

Comment:
Isn't it better to preserve the original signature order? (i.e. keep the event handler arg first).

Reply:
Actually I did this to be consistent with the signature of the other methods in this class. For example, notice how adding a `TestPlatformOptions` parameter always keeps the events handler last:
```
public void RunTests(
    IEnumerable<TestCase> testCases,
    string runSettings,
    ITestRunEventsHandler testRunEventsHandler)
```
```
public void RunTests(
    IEnumerable<TestCase> testCases,
    string runSettings,
    TestPlatformOptions options,
    ITestRunEventsHandler testRunEventsHandler)
```

On top of this, I'd rather not go through rewiring the logic and fixing the tests if there's no immediate benefit to be gained.